### ISSUE TITLE
configrationPolicies missing from API?

### ISSUE THREAD

**Issue by Kvikku** [microsoftgraph/microsoft-graph-docs-contrib#53]
I am trying to query configurationPolicies in graph explorer as described in this article, however the configurationPolicies node does not exist. 

According to the [documentation ](https://learn.microsoft.com/en-us/graph/api/intune-deviceconfigv2-devicemanagementconfigurationpolicy-list?view=graph-rest-beta#http-request)the query should be like this:

GET /deviceManagement/configurationPolicies/{deviceManagementConfigurationPolicyId}

See this image for an example of what I am seeing on my end:

![image](https://github.com/microsoftgraph/microsoft-graph-docs/assets/85877485/2c9f1842-1acb-4b13-8064-7fe73ce425ce)

Is this a known bug?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1bafe8ec-22f9-3412-b47c-318c7e29f90c
* Version Independent ID: 52246316-7f3c-efea-b79c-669d650746bd
* Content: [List deviceManagementConfigurationPolicies - Microsoft Graph beta](https://learn.microsoft.com/en-us/graph/api/intune-deviceconfigv2-devicemanagementconfigurationpolicy-list?view=graph-rest-beta#http-request)
* Content Source: [api-reference/beta/api/intune-deviceconfigv2-devicemanagementconfigurationpolicy-list.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/intune-deviceconfigv2-devicemanagementconfigurationpolicy-list.md)
* Product: **intune**
* Technology: **microsoft-graph**
* GitHub Login: @jaiprakashmb
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by Mike-Crowley** [comment]
Are you using the beta api? I believe this is currently required.

**Comment by Kvikku** [comment]
Yes, I am using beta API.

It seems to be an issue with graph explorer auto complete functionality when you are typing in the address bar. 

When typing, it doesn't recognize the "configurationPolicies" part of the query, which initiality made me think something was wrong. But if you just hit "run query" after typing the full API path , the result is correct. I was able to both GET and POST 😊

I've tried on multiple browsers, multiple tenants and in incognito mode. The behaviour is the same on my computer. I also asked a colleague of mine to check, and he experienced the same.
